### DISCUSSION TITLE
Technical question

### DISCUSSION THREAD

**Discussion by wwieder** [NCAR/LMWG_dev#32] [General]
It seems like it may be helpful to have a discussion thread on technical questions?  I'll start this thread here for quick questions we can hopefully address without too much trouble.

**Comment by wwieder** [comment]
### What's happened to observations on some of the regional plots of monthly climatologies?

For example recent cases, [have few or no observations](https://webext.cgd.ucar.edu/I1850/ctsm51_cesm23a16bctsm51d130_ne30pg3ne30pg3mg17_GSWP3V1_ABCD_1850/lnd/ctsm51_cesm23a16bctsm51d130_ne30pg3ne30pg3mg17_GSWP3V1_ABCD_1850.11_20-ctsm51_cesm23a16bctsm51d130_ne30pg3ne30pg3mg17_GSWP3V1_1850.11_20/set3/set3_landf_Polar.png), but this wasn't always the case ([e.g. CLM5](https://webext.cgd.ucar.edu/I20TR/clm50_r270_1deg_GSWP3V1_isofix_newpopd_hist/lnd/clm50_r270_1deg_GSWP3V1_isofix_newpopd_hist.1995_2014-b.e21.BHIST.f09_g17.CMIP6-historical.001.1995_2014/set3/set3_landf_Polar.png))?  

Is this just because the time period of the initial conditions and observations don't align? @olyson @slevis-lmwg

**Comment by olyson** [comment]
Like I think you were, after the CESM project meeting, I was prompted to look at that by the discussion!  It looks like observations were dropped from the set_3 plots when I put in the requested soil moisture storage annual cycle which doesn't have observations.  I've put in a fix for that now and am now getting obs on those plots.  I'm rerunning the diagnostics for the fully coupled simulations (46, 47, 52, 53, 54) and will post those.  For the land-only simulations, I think I'll rerun the diagnostics for the 1850 ABCD and the Sturm historical.  Let me know if you'd like reruns of anything else.

**Comment by olyson** [comment]
The diagnostics noted above have been updated.
